libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/maneuver/ManeuverView.java

```diff
@@ -234,9 +234,12 @@ protected void onDraw(Canvas canvas) {
     if (maneuverViewUpdate != null) {
       maneuverViewUpdate.updateManeuverView(canvas, primaryColor, secondaryColor, size, roundaboutAngle);
     }
-    if (drivingSide.equals(STEP_MANEUVER_MODIFIER_RIGHT)) {
-      boolean flip = SHOULD_FLIP_MODIFIERS.contains(maneuverModifier);
-      setScaleX(flip ? -1 : 1);
+    boolean flip = SHOULD_FLIP_MODIFIERS.contains(maneuverModifier);
```

Comment:
is this equivalent to:
```suggestion
    boolean flip = SHOULD_FLIP_MODIFIERS.contains(maneuverModifier);
    if (STEP_MANEUVER_MODIFIER_LEFT.equals(drivingSide) && STEP_MANEUVER_MODIFIER_UTURN.contains(maneuverModifier)) {
        setScaleX(flip ? 1 : -1);
    } else {
       setScaleX(flip ? -1 : 1);
    }
```